handlers/istio_config.go

```diff
@@ -121,14 +121,43 @@ func IstioConfigDetails(w http.ResponseWriter, r *http.Request) {
 	RespondWithJSON(w, http.StatusOK, istioConfigDetails)
 }
 
+func IstioConfigDelete(w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+	namespace := params["namespace"]
+	objectType := params["object_type"]
+	object := params["object"]
+
+	api := business.GetIstioAPI(objectType)
+	if api == "" {
+		RespondWithError(w, http.StatusBadRequest, "Object type not managed: "+objectType)
+		return
+	}
+
+	// Get business layer
+	business, err := business.Get()
+	if err != nil {
+		RespondWithError(w, http.StatusInternalServerError, "Services initialization error: "+err.Error())
+		return
+	}
+
+	err = business.IstioConfig.DeleteIstioConfigDetail(api, namespace, objectType, object)
```

Comment:
Question, if object doesn't exist. This will send an OK or an 404 ?

Reply:
I mean, perhaps it's just a matter to check if there is an 404 in the err that comes from the underlying API and just change this to not return always a 500.

Reply:
you're right i'll check that